.eslintrc.cjs

```diff
@@ -11,7 +11,8 @@ module.exports = {
   parserOptions: { ecmaVersion: 'latest', sourceType: 'module' },
   plugins: ['react-refresh'],
   rules: {
-    'react-refresh/only-export-components': 'warn',
+    'no-nested-ternary': 'warn',
```

Comment:
why editing eslint rules?

Reply:
'react-refresh/only-export-components': 'warn', 
there is something wrong with this rule maybe a syntax one!

Reply:
'no-nested-ternary': 'warn',

Reply:
and actually you turned off this rule
```'react-refresh/only-export-components': 'warn',```